src/main/resources/db/migration/nomis/data/R__4_27__LIVING_UNITS_MV.sql

```diff
@@ -1,18 +1,45 @@
 INSERT INTO LIVING_UNITS_MV (AGY_LOC_ID, ROOT_LIVING_UNIT_ID, LIVING_UNIT_ID, PARENT_LIVING_UNIT_ID, TRACKING_FLAG, DESCRIPTION) values ('LEI', -1,  -1,  null, 'N', 'LEI-A');
 INSERT INTO LIVING_UNITS_MV (AGY_LOC_ID, ROOT_LIVING_UNIT_ID, LIVING_UNIT_ID, PARENT_LIVING_UNIT_ID, TRACKING_FLAG, DESCRIPTION) values ('LEI', -1,  -2,    -1, 'N', 'LEI-A-1');
+INSERT INTO LIVING_UNITS_MV (AGY_LOC_ID, ROOT_LIVING_UNIT_ID, LIVING_UNIT_ID, PARENT_LIVING_UNIT_ID, TRACKING_FLAG, DESCRIPTION) values ('LEI', -2,  -2,    -1, 'N', 'LEI-A-1');
```

Comment:
There were many row missing from this table which exists in real database, basically a location has a de-normalised list linking to each parent